src/java/Computer.java

```diff
@@ -0,0 +1,48 @@
+package java;
+
+import java.util.Scanner;
+
+public class Computer {
+    protected boolean processor;
+    protected boolean ram;
+    protected boolean harddrive;
+    protected int cikl;
```

Comment:
@AnnaNikolya имена переменных транслитом фуфу
```suggestion
    protected boolean cpu;
    protected boolean ram;
    protected boolean hardDrive;
    protected int loop;
```